bokeh/plotting/helpers.py

```diff
@@ -247,6 +247,36 @@ def _tool_from_string(name):
 
         raise ValueError("unexpected tool name '%s', %s tools are %s" % (name, text, nice_join(matches)))
 
+
+def _process_axis_and_grid(plot, axis_type, axis_location, minor_ticks, axis_label, rng, dim):
+    axiscls = _get_axis_class(axis_type, rng)
+    if axiscls:
+
+        if axiscls is LogAxis:
+            # TODO (bev) this mapper type hinting is ugly
+            if dim == 0:
+                plot.x_mapper_type = 'log'
+            elif dim == 1:
+                plot.y_mapper_type = 'log'
+            else:
+                raise ValueError("received invalid dimension value: %r" % dim)
+
+        # this is so we can get a ticker off the axis, even if we discard it
+        axis = axiscls(plot=plot if axis_location else None)
+
+        if isinstance(axis.ticker, ContinuousTicker):
+            axis.ticker.num_minor_ticks = _get_num_minor_ticks(axiscls, minor_ticks)
+
+        axis_label = axis_label
```

Comment:
Well, harmless at least. :) Thanks for the catch I will remove in some other PR soon